media/css/firefox/new/scene1_gb_de.scss

```diff
@@ -0,0 +1,207 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+@import "../../pebbles/includes/lib";
+
+/* -------------------------------------------------------------------------- */
+// Custom key features section
+
+.features {
+  text-align: center;
+  background: #fff;
+
+  .features-title {
+      color: #000;
+      margin-bottom: 60px;
+  }
+
+  h3 {
+    @include font-size-level4;
+    color: $color-quantum-blue;
+    font-weight: normal;
+    line-height: 1.3;
+    margin: 20px 0 20px 0;
+  }
+
+  ul {
+    margin: 0;
+
+    li {
+      background-position: 50% 20%, 50% 5%;
+      background-repeat: no-repeat;
+      padding-top: 180px;
+
+      &.private {
+        background-image: url("/media/img/firefox/new/icon-mask.svg"), url("/media/img/firefox/new/blob.svg");
+      }
+
+      &.faster {
+        background-image: url("/media/img/firefox/new/icon-clock.svg"), url("/media/img/firefox/new/blob2.svg");
+      }
+
+      &.tracking {
+        background-image: url("/media/img/firefox/new/icon-protection.svg"), url("/media/img/firefox/new/blob3.svg");
+      }
+
+      &.password {
+        background-image: url("/media/img/firefox/new/icon-password.svg"), url("/media/img/firefox/new/blob2.svg");
+      }
+
+      &.sync {
+        background-image: url("/media/img/firefox/new/icon-sync.svg"), url("/media/img/firefox/new/blob.svg");
+      }
+
+      &.switch {
+        background-image: url("/media/img/firefox/new/icon-switch.svg"), url("/media/img/firefox/new/blob3.svg");
+      }
+
+      &:last-child p {
+        margin-bottom: 20px;
+      }
+    }
+  }
+
+  p {
+    @include font-size-level5;
+    margin: 0 auto 40px;
+    max-width: 16em;
+  }
+
+  @media #{$mq-tablet} {
+    padding-top: 40px;
+
+    ul {
+      @include flexbox;
+      flex-wrap: wrap;
```

Comment:
In IE 10 the features seem to be breaking out the page:

<img width="1401" alt="Screenshot 2019-03-19 at 09 35 15" src="https://user-images.githubusercontent.com/400117/54595435-814e8780-4a2a-11e9-88db-3f9eabeba0f5.png">

Using the `flex-wrap` mixin seems to fix things